ceph.yaml

```diff
@@ -134,7 +135,7 @@ pipeline:
 
   - uses: patch
     with:
-      patches: fix-osd-class-dir.patch
+      patches: fix-osd-class-dir.patch wolfi-update.patch update-podman.patch
```

Comment:
is it possible to add test in package level to ensure the intended functionality of this patch works?